app/src/androidTest/java/ch/epfl/qedit/edit/EditOverviewFragmentTest.java

```diff
@@ -22,6 +26,21 @@
     public final FragmentTestRule<?, EditOverviewFragment> testRule =
             FragmentTestRule.create(EditOverviewFragment.class);
 
+    @Rule
```

Comment:
How come that you added these lines but the tests themselves didn't change? Perhaps I'm missing something here.